packs/dotnet/dotnet/dashboards/dotnet.json

```diff
@@ -0,0 +1,585 @@
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 7
+          },
+          "title": "VM Overview",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 2594082,
+                "query": "FROM Metric SELECT average(apm.service.cpu.usertime.utilization) * 100 as 'Average CPU Utilization ', average(apm.service.memory.physical) as 'Average Physical Memory' WHERE appName = 'DotNetApplication' SINCE 1 WEEK AGO"
+              }
+            ],
+            "thresholds": []
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.area"
+          },
+          "layout": {
+            "column": 8,
+            "row": 1,
+            "height": 3,
+            "width": 5
+          },
+          "title": "CPU Utilization",
+          "rawConfiguration": {
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 2594082,
+                "query": "SELECT rate(sum(apm.service.cpu.usertime.utilization), 1 second) * 100 as cpuUsage FROM Metric WHERE appName = 'DotNetApplication' SINCE 3600 seconds AGO TIMESERIES"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.area"
+          },
+          "layout": {
+            "column": 1,
+            "row": 4,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Memory Heap Used",
+          "rawConfiguration": {
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 2594082,
+                "query": "SELECT average(`apm.service.memory.heap.used`) as 'Memory Heap Used' FROM Metric WHERE appName LIKE '%' SINCE 30 MINUTES AGO TIMESERIES"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.area"
+          },
+          "layout": {
+            "column": 5,
+            "row": 4,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Average Memory Heap Used Based On Instance",
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 2594082,
+                "query": "SELECT average(`apm.service.memory.heap.used`) FROM Metric WHERE appName LIKE 'JavaApp' facet instanceName SINCE today TIMESERIES"
```

Comment:
same comment: update the WHERE clause so it works for every customer (JavaApp is specific to your app I assume)